util/util-internal-archive-layout/src/layout.ts

```diff
@@ -444,4 +444,5 @@ function* pack<T>(items: T[], size: number): Iterable<T[]> {
     }
 
     items.splice(0, offset)
+    yield items
```

Comment:
but if we keep this code as it is then some chunks will not be complete. 
recently i discovered that solana raw dataset had ~5 blocks gap. 2 dump isntances were running: one from a specific block and without `--last-block` and another with `--last-block`. so that one with `--last-block` instead of writing a final chunk from 5 blocks were finishing its process without writing anything at all. 
and it happend only because the dumper restarted before this final chunk otherwise it still would be able to write this final chunk

Reply:
The bug is not here. It's just that, blocks left unwritten after the end of ingest loop, should be saved - https://github.com/subsquid/squid-sdk/blob/ea5b017b92be79432a526ec3ec1b9a848e05e98b/util/util-internal-archive-layout/src/layout.ts#L261